README.md

```diff
@@ -113,14 +94,9 @@ class MyElement extends Polymer.Element {
 Define a Polymer property.
 
 `options` is a [Polymer property
-options](https://www.polymer-project.org/2.0/docs/devguide/properties) object.
+options](https://www.polymer-project.org/3.0/docs/devguide/properties) object.
 All standard options are supported, except for `value`; use a property
-initializer instead.
-
-If the [Metadata Reflection API](#metadata-reflection-api) is configured, the
-`type` option (which determines how Polymer de-serializes Element attributes
-for this property) will be inferred from the TypeScript type and can be
-omitted.
+initializer instead. `type` is required.
```

Comment:
Add the following note to make this a little bit more clear: 

> Note that the `type` should be a JavaScript constructor like the Polymer types, not the TypeScript type

Reply:
Done.